docs/configuration/config-file-reference.md

```diff
@@ -975,6 +975,28 @@ lifecycler:
 # After what time a series is considered to be inactive.
 # CLI flag: -ingester.active-series-metrics-idle-timeout
 [active_series_metrics_idle_timeout: <duration> | default = 10m]
+
+global_limits:
```

Comment:
We already have `-ingester.max-global-series-per-user` (`limits config`) right? And if I recall correct in that case global refers to a scope that spans multiple ingesters (and not tenants).

What do you think about renaming this to something more expressive like `cross_tenant_limits` to avoid confusion between these options?

Reply:
I agree on this feedback. I find the usage of "global" terminology misleading. I'm wondering if we should just call it "instance limits", because are limits applied to each ingester instance.

Reply:
> And if I recall correct in that case global refers to a scope that spans multiple ingesters (and not tenants).

That's a good point!

> I'm wondering if we should just call it "instance limits", because are limits applied to each ingester instance.

Let's use "instance limits".